practice/pig-latin.py

```diff
@@ -3,18 +3,32 @@
 # setup
 VOWELS = ['a', 'e', 'i', 'o', 'u']
 
-# input
-word_to_translate = input('Please give me a word to translate into pig latin: ')
-word_to_translate_lower = word_to_translate.lower()
-word_to_translate_list = word_to_translate_lower.strip()
+def word_to_translate_from_user():
+    """Asks user for a word, converts to lower case, removes puncuation.
+    """
```

Comment:
[PEP-257](https://www.python.org/dev/peps/pep-0257/) says single line docstrings have closing quotes on the same line.